driver/kickoff.md

```diff
@@ -1,6 +0,0 @@
-# Creating a Driver
```

Comment:
why we are removing this? I really thing that first we should explain, is the architecture and the constraints without using examples. An example is a indirect way to document something and once you already know how to use it, is hard to read to verify something